server/repository/src/migrations/mod.rs

```diff
@@ -60,6 +62,7 @@ pub fn migrate(
         Box::new(V1_01_01),
         Box::new(V1_01_02),
         Box::new(V1_01_03),
+        Box::new(V1_01_04),
     ];
```

Comment:
This migration is not needed because you already did the diesel migration. Could you please remove to avoid future merge conflicts with other 1.01.04 migrations?

Reply:
Ok, will do. 
I wasn't recreating my database every time, as that's tedious, particularly when reviewing PRs - but now have scripted and it's fairly quick.